### ISSUE TITLE
DOES NOT BUILD: missing Codality Inc developer identity

### ISSUE THREAD

**Issue by joelpresence** [Automattic/simplenote-macos#113]
I followed all instructions, updated cocoapods, pod install, etc.  Then when I try to build I get:

```
Developer ID Application: Codality, Inc.: no identity found
Developer ID Application: Codality, Inc.: no identity found
Developer ID Application: Codality, Inc.: no identity found
Developer ID Application: Codality, Inc.: no identity found
Command /bin/sh failed with exit code 1
```
 ... and the build fails.

What is Codality and why is this hard-coded into the project?

I tried googling but cannot find anything about Codality.  The build is broken ... :-(

I am using Xcode 8.2.1 (8C1002) on macOS 10.12.2 using ruby 2.2.2p95 (2015-04-13 revision 50295) [x86_64-darwin14] and cocoapods 1.1.1.

Any ideas?

**Comment by joelpresence** [comment]
As per https://www.lantean.co/sparkle-in-sandbox-mode/, you're supposed to set the developer identity in the run script build phase, but it's hard-coded to Codality Inc and I don't see any instructions in the Wiki that this needs to be changed.  So could we add instructions about how to change this (I can help if desired) or not hard code this and instead use a build config variable?

Thanks!

**Comment by joelpresence** [comment]
And I'm not trying to sound harsh, I totally appreciate that you released this.  :-)  I'm just frustrated.

**Comment by jleandroperez** [comment]
Hello @joelpresence,

Not sounding harsh at all! welcome to the awesome world of macOS development!!. The web is, apparently, an extremely small place. The link you've posted [in this comment](https://github.com/Automattic/simplenote-macos/issues/113#issuecomment-273602806) leads to... actually... my personal blog (what are the odds!?).

Reason why you're having issues building, is because you need to setup your own Apple's Developer Certificates (+ Update the project settings). 

Codesign in the Mac works exactly the same way as it does in iOS: Apple has in place several mechanisms to warrantee that the binary (A) comes from a trusted developer, (B) hasn't been tampered with, and (C) is running on an authorized device.

[You may find detailed documentation here](https://developer.apple.com/library/content/documentation/IDEs/Conceptual/AppDistributionGuide/MaintainingCertificates/MaintainingCertificates.html).

Long story short, you'll need to:
- Signup for an Apple Developer Account
- Create a Developer's Certificate
- Create a Provisioning Profile for the app
- Update the App's Project

I believe the entire process has been simplified in the latest Xcode, however, i haven't tried it myself. You can find a 30' video explaining the [new mechanisms here](https://developer.apple.com/videos/play/wwdc2016/401/).

Alternatively, just for testing purposes, you could just try disabling `App Sandboxing` / `Codesigning` altogether (screenshot attached below!).

Hope that helps!

![codesign](https://cloud.githubusercontent.com/assets/1195260/22102653/1ceafa2a-de30-11e6-8a87-397443869b57.jpg)
